doc/source/whatsnew/v1.4.4.rst

```diff
@@ -9,6 +9,17 @@ including other versions of pandas.
 {{ header }}
 
 .. ---------------------------------------------------------------------------
+.. _whatsnew_150.enhancements:
+
+Enhancements
+~~~~~~~~~~~~
+
+.. _whatsnew_150.enhancements.read_sas-encoding:
+
+read_sas infers encoding from file
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:func:`read_sas` now supports using ``encoding='infer'`` to correctly read and use the encoding specified by the sas file. (:issue:`48048`)
```

Comment:
This should be in `v1.5.0.rst` in the IO section

Reply:
👍 
May I ask why? This is my first pandas contribution, how are the versions planned? This is not a breaking change so I expected it to be included in the next minor.

Reply:
Point releases (1.4.4) are reserved for regression and bug fixes from the prior minor release.

Minor releases (1.5) are reserved for everything else.